src/main/java/uk/gov/hmcts/reform/hmc/repository/HearingResponseRepository.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.hmc.repository;
+
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.reform.hmc.data.HearingEntity;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Transactional(propagation = Propagation.REQUIRES_NEW)
+@Repository
+public interface HearingResponseRepository extends CrudRepository<HearingEntity, Long> {
```

Comment:
```suggestion
public interface HearingResponseRepository extends CrudRepository<HearingResponseEntity, Long> {
```

Reply:
Whoops. Why did the query work while pointing at the wrong entity?